mods/cnc/cursors.yaml

```diff
@@ -173,6 +173,19 @@ Cursors:
 		move-blocked-minimap:
 			Start: 25
 			Length: 1
+		attackmove-blocked:
+			Start: 24
+			Length: 1
+		attackmove-blocked-minimap:
+			Start: 25
```

Comment:
Unless something went wrong with my local branch, this should be 26, while 25 could be used for the large cursor instead of 24.